<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoGenVisualization</class>
 <widget class="QWidget" name="GeoGenVisualization">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PeryiteUI - GeoGen Visualization</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QGraphicsView" name="FlandersMap">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>397</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
  </widget>
  <widget class="QLabel" name="CircleInfoTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>1000</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CircleInfoText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>1000</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
